src/IB/IBFESurfaceMethod.cpp

```diff
@@ -2131,7 +2763,11 @@ IBFESurfaceMethod::getFromRestart()
         TBOX_ERROR(d_object_name << ":  Restart file version different than class version." << std::endl);
     }
     db->getIntegerArray("d_ghosts", d_ghosts, NDIM);
-    d_use_jump_conditions = db->getBool("d_use_jump_conditions");
+    d_use_pressure_jump_conditions = db->getBool("d_use_pressure_jump_conditions");
+    d_use_velocity_jump_conditions = db->getBool("d_use_velocity_jump_conditions");
+    d_compute_fluid_traction = db->getBool("d_compute_fluid_traction");
+    d_traction_interior_side = db->getBool("d_d_traction_interior_side");
```

Comment:
I think that we can remove `d_traction_interior_side`? It's not being used anywhere